tests20/python_client/base/client_request.py

```diff
@@ -121,11 +121,15 @@ def _connect(self):
         res = self.connection.connect(alias='default')
         return res
 
-    def _collection(self, name=None, data=None, schema=None, check_res=None, **kwargs):
+    def _collection(self, name=None, data=None, schema=None, check_res=None, exist=False, multiple=False, **kwargs):
         """ Testing func """
-        self._connect()
+        res, _ = self.connection.get_connection_addr(alias='default')
+        if (not res) or multiple:
+            self._connect()
         name = cf.gen_unique_str("ApiReq") if name is None else name
         schema = cf.gen_default_collection_schema() if schema is None else schema
+        if exist:
```

Comment:
in this case, i think you should call self.collection.collection_init()

Reply:
OK, modified.